### PR TITLE
[chore] remove pjanotti from codeowners

### PR DESCRIPTION
**Pull request by atoulme**
This PR removes @pjanotti from multiple components. Please note that as result, the carbonexporter, carbonreceiver and wavefrontreceiver will no longer have active codeowners and may be therefore be considered unmaintained.

Please see issue #23246 for discussion.

### CODE REVIEW

**Review by dmitryax — APPROVED** [review]

**Comment by dmitryax** [comment]
Waiting for review from @pjanotti

**Review by crobert-1 — APPROVED** [review]

**Review by pjanotti — APPROVED** [review]
Thanks @atoulme - I haven't been able to keep following the collector changes so it doesn't make sense to be still listed as a code owner.

**Pull request merged**
